catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -437,36 +438,42 @@ private static boolean isAttributeMultiValued(String attribute, MetacardImpl met
         return multiValued;
     }
 
-    private static String getPhoneNumberString(TelephoneNumberType digits) {
-        String phoneNumber;
-        if (StringUtils.isBlank(digits.getExtension())) {
-            phoneNumber = String.format("(%s) %s", digits.getAreaCode(), digits.getNumber());
-        } else {
-            phoneNumber = String.format("(%s) %s extension %s",
-                    digits.getAreaCode(),
-                    digits.getNumber(),
-                    digits.getExtension());
-        }
-
-        return phoneNumber;
+    private static String getPhoneNumber(TelephoneNumberType digits) {
+        StringBuilder phoneNumberBuilder = new StringBuilder();
+
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder,
+                "(%s)",
+                digits.getAreaCode(),
+                "");
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder, "%s", digits.getNumber(), " ");
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder,
+                "extension %s",
+                digits.getExtension(),
+                " ");
+        return phoneNumberBuilder.toString();
     }
 
     private static void setMetacardAddressAttribute(List<PostalAddressType> addresses,
             String metacardAttribute, MetacardImpl metacard) {
+
         if (CollectionUtils.isNotEmpty(addresses)) {
+            StringBuilder addressString = new StringBuilder();
             PostalAddressType address = addresses.get(0);
 
-            String metacardAddress = String.format("%s, %s, %s %s, %s",
-                    address.getStreet(),
-                    address.getCity(),
+            addressString = buildNonNullString(addressString, "%s", address.getStreet(), "");
```

Comment:
After our conversation, realized this is more complicated than I thought it was at first. Ignore my comment